README.md

```diff
@@ -46,7 +46,8 @@ The list of items you’ll need is also included below:
 
 ## Setup and configuration
 
-Use of balenaDash requires that you allocate more memory to the GPU. This is achieved by adding (or editing the existing) the **Device configuration variable** `RESIN_HOST_CONFIG_gpu_mem`, for this project we recommend setting it to `396`.
```

Comment:
I don't think we should drop this info entirely without replacing it somewhere else (unless it is somewhere else and I can't find it), as it's still critical information required to run the project.

Reply:
As in, change it to tell the user that the device has had it's GPU memory configuration changed so that this project runs?

Reply:
I'm thinking for the cases where someone doesn't use the button, as I can't imagine everyone will just start using it immediately